cfme/common/physical_server_views.py

```diff
@@ -161,7 +161,17 @@ class PhysicalServerManagePoliciesView(BaseLoggedInPage):
 
     @property
     def is_displayed(self):
-        return False
+        return self.entities.get_id_by_name(self.context["object"].name) is not None
+
+
+class PhysicalServerEditTagView(BaseLoggedInPage):
```

Comment:
Required: Use Taggable class instead https://github.com/ManageIQ/integration_tests/blob/8b25294fc81324e1d4450b7b37d30fe85f19a448/cfme/common/__init__.py#L249, this is a duplication